portfolio/src/main/webapp/script.js

```diff
@@ -374,4 +374,36 @@ function toggleCommentForm(loginStatus) {
 function commentsInit() {
   getComments();
   fetchLoginStatus();
+}
+
+/**
+ * Initializes the map.
+ */
+function mapInit() {
+  const homeIsolabellaCoords = { lat: 44.9067176, lng: 7.9096066 };
+  const universityCoords = { lat: 45.0623602, lng: 7.6599514 };
```

Comment:
Since these are technically constants, you could move them out of the function and use CONSTANT_CASE for the variable name